src/features/retrospective/ui/RetrospectiveSummary.tsx

```
@@ -0,0 +1,94 @@
+      <span className='text-[22px] text-[#222] font-semibold'>
+        {total.toLocaleString()}원
+      </span>
+      <div className='flex flex-row w-full h-6 rounded-[4px] bg-gray-100 mt-4 mb-8 overflow-hidden'>
+        {summaries.map(({ color, percentage }) => (
+          <div
+            style={{
+              width: `${percentage.toString()}%`,
+              backgroundColor: color,
+            }}
+          />
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)